source/hassio/installation.markdown

```diff
@@ -31,12 +31,17 @@ The following will take you through the steps required to install Home Assistant
 
 2. Install Home Assistant:
 
-   - Flash the downloaded image to an SD card using [balenaEtcher][balenaEtcher]. If using a Pi, we recommend at least a 32 GB SD card to avoid running out of space.
-   - Load the appliance image into your virtual machine software. (Note: You are free to assign as much resources as you wish to the VM, please assign enough based on your add-on needs)
+   - **Raspberry Pi, Tinkerboard, or an ODROID**:
```

Comment:
The Intel NUC can also run from SD?

Reply:
Is that a supported scenario? Proposal:
````
Flash the downloaded image to your SSD/hard disk (recommended) or SD-card using...
````

Reply:
Nope, in general, the SSD isn't supported.

Reply:
Changed as discussed on Discord.